pkgs/applications/audio/string-machine/default.nix

```
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, boost, cairo, lv2, pkg-config }:
+
+stdenv.mkDerivation rec {
+  pname = "string-machine";
+  version = "unstable-2019-11-12";
+
+  src = fetchFromGitHub {
+    owner = "jpcima";
+    repo = pname;
+    rev = "609eddf395735853b2ad987864d0b3539e7f83df";
+    sha256 = "1l03m8yl37vs1517w8kc6gml1rwjxcq8ws2whqiwgczksljb4r7s";
+    fetchSubmodules = true;
+  };
+
+  postPatch = ''
+    patchShebangs ./dpf/utils/generate-ttl.sh
+  '';
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [
+    boost cairo lv2
+  ];
+
+  makeFlags = [
+    "PREFIX=$(out)"
+  ];
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/jpcima/string-machine";
+    description = "Digital model of electronic string ensemble instrument";
+    maintainers = [ maintainers.magnetophon ];
+    platforms = platforms.linux;
```

uses x86 specific extensions, please do one of the following
```suggestion
    platforms = intersectLists platforms.linux platforms.x86;
```
or
```suggestion
    platforms = [ "i686-linux" "x86_64-linux" ];
```

---

Done!
Also upped the version.